### PR TITLE
udev: several rule cleanups

### PR DESCRIPTION
**Pull request by yuwata**

### CODE REVIEW

File: rules.d/60-persistent-storage.rules.in [hunk 1]
```diff
@@ -29,42 +29,33 @@ ENV{DEVTYPE}=="partition", \
   IMPORT{parent}="ID_F[!S]*", IMPORT{parent}="ID_F", \
   IMPORT{parent}="ID_FS[!_]*", IMPORT{parent}="ID_FS"
 
+ENV{DEVTYPE}=="partition", ENV{.PART_SUFFIX}="-part%n"
+ENV{DEVTYPE}!="partition", ENV{.part_SUFFIX}=""
```
Comment (poettering) [inline rules.d/60-persistent-storage.rules.in]: accidental lowercasing?

**Comment by poettering** [comment]
I still wished somebody who's into SCSI would take the SCSI stuff.

File: rules.d/60-persistent-storage.rules.in (hunk 1 above)
Comment (yuwata) [reply rules.d/60-persistent-storage.rules.in]: Ouch.

**Comment by yuwata** [comment]
> I still wished somebody who's into SCSI would take the SCSI stuff and maintain it as part of some scsi-specific package.

Yeah, but that has not happened for last ~5 years...

**Comment by yuwata** [comment]
Fixed the issue found by @poettering. PTAL.

**Comment by poettering** [comment]
lgtm now, but needs a rebase

**Pull request merged**